pkg/cli/common.go

```diff
@@ -246,3 +247,15 @@ func VerifyFlagsArgsOrder(args []string) error {
 	}
 	return nil
 }
+
+func GetDefaultAuthFile() string {
```

Comment:
I'm surprised golang didn't yell at you for not having a doc line above this....